rebench/tests/small_with_version_command.conf

```diff
@@ -0,0 +1,33 @@
+# Config file for ReBench
+# Config format is YAML (see http://yaml.org/ for detailed spec)
+
+# this run definition will be chosen if no parameters are given to rebench.py
+default_experiment: Test
+default_data_file:  'small.data'
+
+# general configuration for runs
+runs:
+    invocations:  10
+    retries_after_failure: 3
+
+benchmark_suites:
+    Suite:
+        gauge_adapter: TestExecutor
+        command: TestBenchMarks ~/suiteFolder/%(benchmark)s
+        benchmarks:
+            - Bench1
+            - Bench2
+
+executors:
+    TestRunner1:
+        path: ~/PycharmProjects/ReBench/rebench/tests
+        executable: test-vm1.py %(cores)s
+        cores: [1]
+        version_command: "python --version"
+
```

Comment:
Do we really need three new config files for this? Could this perhaps be solved with having 3 different executors in the same config file and testing directly for those?